agent/src/main/java/com/appland/appmap/config/AppMapConfig.java

```diff
@@ -55,10 +57,13 @@ public static AppMapConfig load(File configFile) {
       return null;
     }
 
-    Yaml yaml = new Yaml();
+    Representer representer = new Representer();
+    representer.getPropertyUtils().setSkipMissingProperties(true);
+    
+    Yaml yaml = new Yaml(new Constructor(AppMapConfig.class),representer);
```

Comment:
You copied the missing space from stack overflow here ;)

Reply:
Yes, yes I did. I've fixed it now, and added a link to the post.